src/test/unit/altitude_hold_unittest.cc.txt

```diff
@@ -91,9 +89,9 @@ TEST(AltitudeHoldTest, IsThrustFacingDownwards)
 
     for (uint8_t index = 0; index < testIterationCount; index ++) {
         inclinationExpectation_t *angleInclinationExpectation = &inclinationExpectations[index];
-#ifdef DEBUG_ALTITUDE_HOLD
-        printf("iteration: %d\n", index);
-#endif
+// #ifdef DEBUG_ALTITUDE_HOLD
+//         printf("iteration: %d\n", index);
+// #endif
```

Comment:
Commented code should be removed

Reply:
that file can be removed.  it's the original test file from the old PR.